src/Microsoft.VisualStudio.ProjectSystem.Managed/Extensions/ProjectConfigurationExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.ProjectSystem.Build;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    internal static class ProjectConfigurationExtensions
+    {
+        /// <summary>
+        /// Returns true if this is a cross-targeting project configuration with a "TargetFramework" dimension.
+        /// This is true for a project with an explicit "TargetFrameworks" property, but no "TargetFrameworkMoniker" or "TargetFramework" property.
```

Comment:
@nguerrera I'll update the comment. What you mentioned here is what our dimension provider actually does.